index-START.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>

```

Comment:
I think you can rename this back to `index.html`?

Reply:
done